cryptofeed/backends/influxdb.py

```diff
@@ -77,7 +90,7 @@ def __init__(self, addr: str, db: str, key=None, create_db=True, numeric_type=st
         d = d[:-1]
 
         update = f'{self.key}-{feed},pair={pair} {d},timestamp={timestamp}'
-        await self.http_write('POST', update)
+        await self.http_write('POST', update, self.headers)
```

Comment:
you don't need to have a class member as a method argument, just access it in the method.

Reply:
In that case, will it better to create `header` inside the each method? 

Both `InfluxBookCallback._write_rows()` and `InfluxCallback.write()` need to pass the header dictionary for authentication when using influx 2.0.

One possible way is to always create the headers object inside the `_write_rows()` and `write()` before `await self.http_write()` method depends on the influx version instead of creating `self.headers`.  

Then, I think we need an initial argument with `InfluxCallback()` indicating whether the user would connect to influx 1.x or 2.0. 

If you give any guidelines or advice then I will modify it.

Reply:
ok, I see what you mean, you'd have to modify http_write in a way thats not ideal, so you should just leave that as-is.